togetherly/pubspec.yaml

```diff
@@ -38,10 +38,16 @@ dependencies:
   envied: ^0.5.3
   provider: ^6.1.1
   supabase_flutter: ^2.0.0
+  sqflite:
+  bloc_test:
+  http: ^1.2.1
```

Comment:
Do you know why these dependencies have been added? (also `sqflite_common_ffi` below)